ASUWebPlatforms/webspark-ci
# [2.15.0] WS2-1783: Added new background options

### Description


#### Solution 
Added new background options

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1783)

### QA Steps

- Pull the branch locally into your existing env.
- Visit an existing Background page.
- Edit the layout of the page, and added a new section.
- Now you can see six new background option

### Checklist


### Verified in browsers 


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
